### ISSUE TITLE
Why Removed on Spigot

### ISSUE THREAD

**Issue by MerlinK11** [dvargas135/eZProtector#73]
Why is that removed on spigot and can you reupload this nice Plugin and Update this to 1.14.+

**Comment by DoNotSpamPls** [comment]
Hi there! eZProtector is now exclusively distributed on the [PaperMC forums](https://papermc.io/forums/t/1-12-2-1-14-x-ezprotector/1361)

**Comment by MerlinK11** [comment]
it is running on Spigot ?

**Comment by DoNotSpamPls** [comment]
Yes, the plugin runs fine on Spigot, although it is recommended that you switch to [Paper](https://papermc.io) for better server performance.

**Comment by MerlinK11** [comment]
i can't use paper whem i use this it is 50% of all my Plugins Broke

**Comment by Matocolotoe** [comment]
Putting this here so that nobody can remove it from my message ❤️
https://github.com/dvargas135/eZProtector/releases/download/4.1.5.1/eZProtector-4.1.5.1.jar

**Comment by DoNotSpamPls** [comment]
@Matocolotoe if you want to use an ancient version of eZProtector, that is on you. 
However, I would not recommend, as there are numerous bugs in the old versions that will never be fixed.
